tools/get_dicom_files.pl

```diff
@@ -90,7 +95,9 @@ =head2 Methods
     ["-d", "string", 1,   \$tmpExtractBaseDir, 
         "base path of the temporary directory where files are extracted"],   
     ["-o", "string", 1,   \$outTarBasename, 
-        "base path of the final .tar.gz file to produce (defaults to 'dicoms')"],   
+        "base path of the final .tar.gz file to produce (defaults to 'dicoms')"], 
+    ["-p", "const" , 1,   \$usePSCID, 
```

Comment:
should it be 'const' or 'bool' here? $usePSCID seems like a boolean to me but I am not sure of the difference between 'const' and 'bool' here.

Reply:
Option `-id` now takes a string argument.